ui/app/com/gu/recipeasy/views/admin/dailybreakdown.scala.html

```diff
@@ -17,7 +17,7 @@
         </thead>
         <tbody>
 
-            @for( dayFigures <- dailyActivityDistribution ){
+            @for( dayFigures <- dailyActivityDistribution.reverse ){
```

Comment:
Um........... That's actually a good question. I will have look and if possible, I will PR the update.